packages/lit-dev-content/site/tutorials/content/reactivity/05.md

```diff
@@ -0,0 +1,45 @@
+For some reactive properties, you may want to customize _change detection_. By default, Lit uses a strict equality comparison to determine whether the new value is different from the old value (`newValue !== oldValue`).
```

Comment:
This is pretty advanced for a beginner tutorial, and I think it can be left out. If you do include it, I think it would be important to explain when to use shouldUpdate v. hasChanged: 
* shouldUpdate: allows you to opt-out of an update based on the full set of pending changes
* hasChanged: allows you to opt-in/out of an update based on a single property being set.

Reply:
I'm not sure if there's enough content to warrant a `Reactivity (intermediate)`

Reply:
For now I removed the `hasChanged()` step and added an `updated()` step instead. Might add an aside to the `shouldUpdated()` step in the future to point to the `hasChanged()` section in the docs... Or maybe not.